src/BridgedRoom.ts

```diff
@@ -640,8 +672,44 @@ export class BridgedRoom {
             return;
         }
         log.debug(`Sending reaction ${reactionKey} for ${event.eventId} as ${ghost.userId}`);
```

Comment:
Would also make this info, and include the channel and ts

Reply:
Added that, made it info on success and warn on failure.